types/index.d.ts

```diff
@@ -71,56 +95,59 @@ export interface PackageJSON {
   bundledDependencies?: string[] | boolean
   bundleDependencies?: string[] | boolean
   config?: Record<string, unknown>
-  contributors?: Contact[] | string[]
+  contributors?: (Contact | string)[]
   cpu?: string[]
   dependencies?: Record<string, string>
   description?: string
   devDependencies?: Record<string, string>
   directories?: Record<string, string>
   engines?: Record<string, string>
   files?: string[]
+  funding: Funding | string | (Funding | string)[]
   homepage?: string
   keywords?: string[]
   license?: string
+  licenses?: DeprecatedLicense | DeprecatedLicense[]
   main?: string
   man?: string | string[]
   name: string
   optionalDependencies?: Record<string, string>
   os?: string[]
+  overrides?: Record<string, Override | string>
   peerDependencies?: Record<string, string>
+  peerDependenciesMeta?: Record<string, PeerDependencyMeta>
   private?: boolean
   publishConfig?: Record<string, unknown>
   repository?: Repository | string
   scripts?: Record<string, string>
+  // https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html#editing-the-packagejson
   types?: string
   version: string
+  workspaces?: string[] | Record<string, string>
 
   [field: string]: unknown
 }
 
+// Note: Contacts (bugs, author, contributors, repository, etc) can be simple
+// strings in package.json, but not in registry metadata.
+
 export interface PackumentVersion extends PackageJSON {
-  // bugs, author, contributors, and repository can be simple strings in
-  // package.json, but not in registry metadata.
-  bugs?: Omit<Contact, 'name'>
+  _hasShrinkwrap?: boolean
+  _id: string
+  _nodeVersion?: string // optional (ref: not defined in uuid@1.4.0)
+  _npmUser?: Contact
+  _npmVersion: string
   author?: Contact
-  // ref: Record type found in uuid@1.4.1 et al
-  browser?: Record<string, string>
+  browser?: Record<string, string> // ref: Record type found in uuid@1.4.1 et al
```

Comment:
```suggestion
  browser?: string | Record<string, string> // ref: Record type found in uuid@1.4.1 et al
```

I believe this can also be a string

https://gist.github.com/defunctzombie/4339901/49493836fb873ddaa4b8a7aa0ef2352119f69211#spec